package pl.merito.shopai.util.exception;

public class UserNotFoundException extends RuntimeException{

}
